front/app/containers/IdeasNewPage/WithJSONForm/index.tsx

```diff
@@ -115,10 +111,36 @@ const IdeasNewPageWithJSONForm = ({ params }: WithRouterProps) => {
     });
     const ideaId = idea.data.id;
 
-    clHistory.push({
-      pathname: `/ideas/${idea.data.attributes.slug}`,
-      search: `?new_idea_id=${ideaId}`,
-    });
+    // Check ParticipationMethodConfig for form submission action
+    if (
+      project?.attributes.process_type === 'timeline' &&
+      !isNilOrError(phases)
+    ) {
+      // Check if URL contains specific phase_id
+      const queryParams = new URLSearchParams(window.location.search);
+      const phaseIdFromUrl = queryParams.get('phase_id');
+
+      const participationMethodFromURL = phases
```

Comment:
Minor: We could do something like this

```
      const phaseUsed = phases.find((phase) => phase.id === phaseIdFromUrl) || getCurrentPhase(phases)
            if (!isNilOrError(phaseUsed)) {
        getMethodConfig(phaseUsed.attributes.participation_method).onFormSubmission(
          project,
          ideaId,
          idea,
          phaseUsed.id
        );
      }
```

I'm using `phaseUsed.id` as I imagine we are trying pass the id from which ever phase picked

We avoid the filter since we need only one phase and this best communicates that. It also combines that way we are getting the phase and can uniformly act on the phase irrespective where we've gotten it from